### PR TITLE
OMERO-server and OMERO-web configuration improvements

### PR DESCRIPTION
**Pull request by sbesson**
Backports a few changes implemented on https://merge-ci.openmicroscopy.org/jenkins/

- recreates a fresh virtual environment for the server and web deployments similarly to what is done #169 
- replace all hardcoded `snoopycrimecop` URLs using `$SPACE_USER` - there should be no more hardcoded user in the configuration jobs

### CODE REVIEW

**Comment by pwalczysko** [comment]
The changes mentioned here were probably done at the time of see https://openmicroscopy.slack.com/archives/D0K9S2VS8/p1562593822023800 please. There is a trello card link which seems to be the only open question, see https://trello.com/c/uyit91ic/65-react-mixed-content - not sure whether this should be solved in this PR though, probably not cc @will-moore

**Comment by joshmoore** [comment]
Also not sure all of these are for general consumption.

**Comment by sbesson** [comment]
Trying to separate the various changes, I think the following two are probably good for general consumption:

- virtualenv -> python -m venv
- snoopycrimecop -> $SPACEUSER

The following change require some information and/or sign-off from the people who made them:

- extra `kill` statements when restarting web
- `omero-figure` installation - I assume it's in the context of https://github.com/ome/omero-figure/pull/366 and not ready
- `parade-crossfilter`  - assuming this is also experimental and not ready
- `omero.web.gallery.category_queries`
- `omero.web.feedback` disabling

**Comment by joshmoore** [comment]
From the questionable list, the only one I can vouch for is the `kill` statement.

**Comment by will-moore** [comment]
I don't think the figure, parade or gallery configs need to be back-ported, thanks.
I don't know why feedback was disabled. Maybe because we'd submit any issues manually to relevant PRs? But it also means that feedback itself is not tested.

File: home/jobs/OMERO-web/config.xml [hunk 1]
```diff
@@ -187,8 +201,8 @@ omero config set omero.web.public.password public
 omero config set omero.web.public.url_filter &apos;^/&apos;
 
 # QA feedback configuration
-omero config set omero.web.feedback.comment.enabled false
-omero config set omero.web.feedback.error.enabled false
+# omero config set omero.web.feedback.comment.enabled false
+# omero config set omero.web.feedback.error.enabled false
```
Comment (manics) [inline home/jobs/OMERO-web/config.xml:205]:
```suggestion
```
Presumably this was a temporary change for testing https://github.com/ome/openmicroscopy/pull/5957 that was never properly reverted?

**Comment by joshmoore** [comment]
> I don't think the figure, parade or gallery configs need to be back-ported, thanks.

i.e. they need reverting in merge-ci, no?

File: home/jobs/OMERO-web/config.xml (hunk 1 above)
Comment (sbesson) [reply home/jobs/OMERO-web/config.xml:205]: So you'd be in favor of re-enabling in merge-ci and latest-ci ?

Comment (manics) [reply home/jobs/OMERO-web/config.xml:205]: Definitely

**Comment by will-moore** [comment]
I guess you could remove the parade-crossfilter config.
The figure config is needed as long as https://github.com/ome/omero-figure/pull/366 is open. I'd like to make some progress on that as it's useful (currently updates figure script automatically etc) but the PR needs more scrutiny and discussion.
Gallery config is needed for testing of the categories(idr) UI. e.g. for https://github.com/ome/omero-gallery/pull/68

**Comment by joshmoore** [comment]
Would it be easiest for you, @willmoore, (him, @seb) to modify merge-ci as you'd expect then we can take a look at the diff again?

**Comment by sbesson** [comment]
Thanks for reviewing the job config and marking the experimental changes @will-moore. Force-pushed the branch to only inlcude the set of changes ready for inclusion in the main devspace repo.

**Review by joshmoore — APPROVED** [review]
Changes look good to me.

**Pull request merged**